src/hotspot/share/compiler/compileTask.hpp

```diff
@@ -218,6 +218,9 @@ class CompileTask : public CHeapObj<mtCompiler> {
   }
   static void  print_ul(const nmethod* nm, const char* msg = nullptr);
 
+  /**
+   * @deprecated Please rely on Compile::inline_printer. Do not directly write inlining information to tty.
```

Comment:
Can we get rid of these instead of making them deprecated?

Reply:
I would definitely like to get rid of this one but it's used in print_trace_type_profile to also print to unified logging and I do not want to touch this as part of this refactoring to keep the changes manageable.